syft/frameworks/torch/mpc/falcon/falcon_helper.py

```diff
@@ -0,0 +1,135 @@
+import torch
+import syft
+
+from syft.frameworks.torch.tensors.interpreters.replicated_shared import ReplicatedSharingTensor
+
+from typing import Union
+
+
+class FalconHelper:
+    XOR_EXPECTED_RING_SIZE = 2
```

Comment:
why is this a global variable thought it's only in xor?

Reply:
Hmm..this is not a global variable, it is accessed only using the class, but I think you are right, we can move it inside the xor method